doc/guides/dts/index.rst

```diff
@@ -346,44 +346,534 @@ Below is sample header content generated for this node, in
    #define DT_INST_0_NXP_KINETIS_SIM_LABEL             DT_NXP_KINETIS_SIM_40047000_LABEL
    #define DT_INST_0_NXP_KINETIS_SIM                   1
 
-Generated macro names follow the format ``DT_<node>_<property>``. For
-``DT_NXP_KINETIS_SIM_40047000_BASE_ADDRESS``, the node part is
-``NXP_KINETIS_SIM_40047000``, based on the compatible string that matched a
-binding for the node (``nxp,kinetis-sim``) and the node's unit address
-(``...@4004700``).
+All macros generated from devicetree start with ``DT_`` and use all-uppercase.
+Most macro names contain a node identifier, and macros that correspond to
```

Comment:
is this statement too generic? isn't "node identifier" just too generic? is it a phandle, a compatible, an instance?

Reply:
I see the next section expands on this quite a bit, fair enough.